.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Resource name and link
+
+<!-- Add the resource name and the link in this section -->
+
+<!-- eg. [Resource Name](link) -> Use this format(including the square and round brackets) to add name link of resource -->
+
+## Short description
+
+<!-- Include a short description of the resource in this section -->
+
+## Why do you think this is a good resource?
+
+<!-- Add a reason that why the resource can be helpful to the community -->
```

Comment:
I think "that" can go off from here. _Add a reason why the resource can be helpful to the community_ reads better.